temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
Avoid using print statements for logging. Use the `logging` module to provide better control over logging levels and output destinations.
```
import logging

logging.basicConfig(level=logging.INFO)

logger = logging.getLogger(__name__)

logger.info('hello')
```